eclair-core/src/main/resources/reference.conf

```
@@ -77,13 +79,23 @@ eclair {
   fee-base-msat = 1000
   fee-proportional-millionths = 100 // fee charged per transferred satoshi in millionths of a satoshi (100 = 0.01%)
 
-  // maximum local vs remote feerate mismatch; 1.0 means 100%
-  // actual check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
-  max-feerate-mismatch = 1.56 // will allow remote fee rates up to 8x bigger or smaller than our local fee rate
+  on-chain-fees {
+    // number of blocks to target when computing fees for each transaction type
+    fee-targets {
```

```suggestion
    target-blocks {
```

And this should be made consistent with `default-feerates.delay-blocks`. Btw why aren't default feerates under the `on-chain-fees` section?

---

Right, `default-feerates.delay-blocks` should go under under the `on-chain-fees` section too